core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java

```diff
@@ -280,36 +287,14 @@ public void doCreateAccountByAdmin(StaplerRequest req, StaplerResponse rsp) thro
      * This can be run by anyone, but only to create the very first user account.
      */
     public void doCreateFirstAccount(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException {
-        boolean inSetup = !Jenkins.getInstance().getInstallState().isSetupComplete();
```

Comment:
... this method was simply reverted to what it was before any of my 'security' changes